components/contract-components/contract-table/cells/released-by.tsx

```diff
@@ -26,8 +26,17 @@ export const ContractReleasedByCell: React.FC<
       .publisher;
 
   const releaserEnsQuery = ens.useQuery(releaser);
+
+  console.log({ isSuccess: fullPublishMetadata, isPrebuilt });
+
   return (
-    <Skeleton isLoaded={fullPublishMetadata.isSuccess || isPrebuilt}>
+    <Skeleton
+      isLoaded={
+        fullPublishMetadata.isSuccess ||
+        fullPublishMetadata.isStale ||
```

Comment:
why does the `isStale` get hit?

Reply:
so we never get fullPublishMetadata if the contract has never been released before, is there a better way to check this?

Reply:
ah, as in we never try to query it or we try to query but it returns a 404?

stale => there is data in the cache but it needs to be re-fetched (which seems like the wrong thing to check)

we could check isIdle *maybe*? or if we try to query and it's a 404 we can check the error?

Reply:
I just updated to check that we're not fetching, this works better I can try again with isIdle see if that works better, I'll brb in 30 mins